packages/venia-concept/src/components/FilterModal/constants.js

```diff
@@ -0,0 +1,24 @@
+export const filterModes = {
+    default: 'default',
+    swatch: 'swatch'
+};
+
+export const filterLayouts = {
+    grid: 'grid',
+    list: 'list'
+};
+
+export const filterRenderOptions = {
+    fashion_color: {
+        mode: filterModes.swatch,
+        options: {
+            layout: filterLayouts.grid,
+            generateColor: true,
+            searchable: true
```

Comment:
Is this another prop that could be removed when provided by the GQL? Maybe we just do away with it and assume always searchable until then.

Reply:
I'm not sure if this is going to be returned by GraphQL out of the box. Searching through filters is not default Magento 2 functionality as far as I'm aware of. According to the design, it's only for color filters: https://magento.invisionapp.com/share/S2Q41VCR45C#/screens/341558811

Reply:
Oh, I didn't even notice that this was a prop on `fashion_color`. I'm cool with this :)